examples/07_customers/07_recover_customer/query.graphql

```diff
@@ -0,0 +1,14 @@
+# This mutation will send a email to customer to reset password of the customer account.
+# Note that this mutation is throttled by IP.
```

Comment:
I'd suggest include a point in comments about including [the Shopify-Storefront-Buyer-IP](https://shopify.dev/api/usage/authentication#optional-ip-header) (https://shopify.dev/api/storefront/2023-01/mutations/customerrecover#:~:text=you%20can%20provide%20a%20Shopify%2DStorefront%2DBuyer%2DIP%20instead%20of%20the%20request%20IP.), totally up to you though.